From e908a3ad04bb40e425542b85fbb3a1eb5a38e194 Mon Sep 17 00:00:00 2001
From: Tee-Kiah Chia <tkchia@users.noreply.github.com>
Date: Thu, 27 Feb 2020 00:52:05 +0800
Subject: [PATCH] [cpp] update token1.tok to make new gperf happy; regen.
 token1.h

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
Upstream-Status: Pending [it was submitted in https://github.com/jbruchon/dev86/pull/19
but then closed by author]

Remove regenerated token1.h as we'll force it to regenerate in do_compile.

---
 cpp/token1.h   | 170 +++++++++++++++++++++++++++++++++----------------
 cpp/token1.tok |   2 +-
 2 files changed, 115 insertions(+), 57 deletions(-)

diff --git a/cpp/token1.tok b/cpp/token1.tok
index cd668ce..a98926f 100644
--- a/cpp/token1.tok
+++ b/cpp/token1.tok
@@ -7,7 +7,7 @@ struct token_trans { char * name; int token; };
 -=,	TK_SUB_ASSIGN
 *=,	TK_MUL_ASSIGN
 /=,	TK_DIV_ASSIGN
-%=,	TK_MOD_ASSIGN
+"%=",	TK_MOD_ASSIGN
 &=,	TK_AND_ASSIGN
 ^=,	TK_XOR_ASSIGN
 |=,	TK_OR_ASSIGN
